### ISSUE TITLE
Setup script fails on Redhat because of typo

### ISSUE THREAD

**Issue by micahlagrange** [awslabs/amazon-kinesis-agent#164]
This error pops up on Redhat systems:

```
install: cannot stat ‘./bin/aws-kinesis-agent.Redhat’: No such file or directory
```

This is because of line 17:

```
  dist=Redhat
```

When the script gets to line 200:

```
  install -m755 ./bin/${daemon_name}.${dist} ${init_dir}/${daemon_name}
```

The filename is actually spelled `aws-kinesis-agent.RedHat` so it obviously doesn't find the file, because it does not exist.

Seems to have changed with this pull request: #152

**Comment by willUrgently** [comment]
See #167

**Comment by micahlagrange** [comment]
resolved by #167
